logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/SequencerMetrics.java

```diff
@@ -28,13 +28,25 @@ final class SequencerMetrics {
           .buckets(1, 2, 3, 5, 10, 25, 50, 100, 500, 1000)
           .labelNames("partition")
           .register();
+
+  private static final Histogram BATCH_LENGTH_BYTES =
```

Comment:
:upside_down_face: Batch_length for the data size for this Batch_size for number of entries. I would expect it the other way. But changing it now could also be confusing.

Reply:
Yeah agreed. We can still change it though. In the end for me it's fine since all of these have help text/descriptions, but I get it :smile: